lib/realtime/data/trip_update.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Realtime.Data.TripUpdate do
+  @moduledoc """
+  The subset of the TripUpdate data in the enhanced-json feed that we use.
+  """
+
+  @type t :: %__MODULE__{
```

Comment:
question: I noticed trip_id is missing here, do we need it, or is this struct typically nested inside another struct that has that information?

Reply:
OK, I thought about this more. So we need the trip ID to connect to OCS trips. But it remains true that we have a `trip` key in `VehiclePosition`, so TripUpdates (predictions) can probably go without it. (We probably want to connect positions to predictions by consist, not trip)